### PR TITLE
Polishing through section 4

### PR DESCRIPTION
**Pull request by flacjacket**
Some fixes for sections 1 - 4, with some wording adjustments and
grammatical fixes.  Also, drops newlines around verbatim blocks that
don't end paragraphs so the following lines are not indented.

### CODE REVIEW

**Comment by asmeurer** [comment]
Thanks. FYI https://github.com/sympy/sympy-paper/pull/124 if you get to the features section.

File: introduction.tex [hunk 1]
```diff
@@ -35,16 +35,16 @@
 import SymPy alongside other Python libraries in their workflow, whether that
 is an interactive workflow or programmatic use as part of a larger system.
 
-SymPy does not have a built in graphical user interface (GUI), however, when
-used in the Jupyter Notebook
-% citation
-SymPy expressions will pretty print using MathJax.
-% citation
```
Comment (asmeurer) [inline introduction.tex:42]: You deleted this comment, but we do need a MathJax citation.

File: introduction.tex [hunk 2]
```diff
@@ -35,16 +35,16 @@
...
+Being developed as a library, SymPy does not have a built-in graphical user
+interface (GUI).  However, SymPy exposes a rich interactive display system,
+including registering printers with Jupyter~\cite{perez2007ipython} frontends,
+including the Notebook and Qt Console, which will pretty print SymPy expressions
+using MathJax or \LaTeX rendering.
```
Comment (asmeurer) [inline introduction.tex]: chktex complains about this. Should be `\LaTeX{}` with the braces.

**Comment by asmeurer** [comment]
Looks good to me. Would merge if Travis would let me.

**Comment by flacjacket** [comment]
Just pushed fixes for those comments, also a few more minor things in `matrices.tex`

**Comment by certik** [comment]
Thanks @flacjacket.

**Pull request merged**